public/schema.json

```diff
@@ -35,6 +35,9 @@
     },
     "/accounts/{account_id}/sites/{site_identifier}/users": {
       "$ref": "refs/paths/accounts_{account_id}_sites_{site_identifier}_users.json"
+    },
+    "/accounts/{account_id}/sites/{site_identifier}/notifiable_users": {
+      "$ref": "refs/paths/accounts_{account_id}_sites_{site_identifier}_notifiable_users.json"
```

Comment:
filename should be `refs/paths/accounts_{account_id}_sites_{site_identifier}_users.json